### ISSUE TITLE
Incorrect Billing Cycle on Manual Invoice

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#299]
In GitLab by @kdjstudios on Apr 6, 2017, 13:29

http://www.servicedesk.answernet.com/profiles/ticket/2017-04-04-72315/conversation

We have submitted a manual draft invoice on account 7755 (It may be T-7755). This invoice when we submitted it showed an invoice date of 4/4/17. Screen shot is attached. However, when the invoice actually went into effect on the account in the billing history page is the 10/28/16, which is of what we believe the system sees would have been the next billing cycle since the account has been inactive for these months.
 

In summary, we need to have this invoice dated for this current billing cycle. That way it will appear on the revenue cycles for this billing cycle. Please make these adjustments.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 6, 2017, 13:32

@smasih @abakshi 

Is this something that can be adjusted easily? If so, may we please have this done by the end of next week.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Apr 7, 2017, 01:33

@kdjstudios 
sure thing, but please review the priority for this.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Apr 15, 2017, 05:33

@kdjstudios 

We were not able to find any invoice, dated **10/28/16** for Chicago --> SABREE MEDICAL SERVICES  ---> T-7755

therefore no action done on this ticket.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 17, 2017, 09:13

@smasih 

My apologies on the typo. This is Invoice 106-8338	10/14/16. 

Moving back to development.

**Comment by kdjstudios** [comment]
In GitLab by @abakshi on Apr 24, 2017, 06:37

Milestone removed

**Comment by kdjstudios** [comment]
In GitLab by @abakshi on Apr 24, 2017, 06:37

@smasih @kdjstudios Moving the milestone to April end.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Apr 29, 2017, 06:32

@kdjstudios 

I am confused..

Invoice **106-8338**	has invoice date **3/1/17**  	  
Invoice **106-7503**	has invoice date of **10/14/16**  

so if I am not wrong we need to put **106-7503** under billing cycle of  **106-8338** which has:- 
* start date: 2017-02-02 (feb 2^nd)
* end date: 2017-03-01 (march 1^st)
* invoice date: 2017-03-01 (march 1^st)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 1, 2017, 08:26

@smasih @abakshi 

Please do not change anything. The 106-8338 had a date of 10/14/16. Someone has already changed this, I presume @abakshi. Please be sure everyone is documenting any changes made and then documenting them in Gitlab as to prevent confusion in the future. 

I have already confirmed with Susan that this is acceptable and completed.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 4, 2017, 07:27

@kdjstudios 

They might have re processed the billing process. @abakshi didn't perform any change on DB level.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 8, 2017, 12:05

@smasih 

Okay, would reprocessing the billing effect a manually drafted invoice? I was under the impression that those were not associated with a billing cycle?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 9, 2017, 01:02

@kdjstudios 

each and every Invoice is connected to a billing cycle, if we generate a manual Invoice... It will fall under Current Billing Cycle.

A billing Cycle can have more than one Invoice for an account.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 9, 2017, 08:14

@smasih 

Thanks for the clarification. I will need to add that to the documentation.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 10, 2017, 08:46

@kdjstudios 

No worries :thumbsup:
